src/functTest/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainerFunctionalTest.groovy

```diff
@@ -353,6 +355,49 @@ class DockerCreateContainerFunctionalTest extends AbstractGroovyDslFunctionalTes
         build('inspectContainer')
     }
 
+    def "Can build an image, create a container and add/drop capabilities"() {
+        given:
+        String containerCreationTask = """
+            task createContainer(type: DockerCreateContainer) {
+                dependsOn pullImage
+                targetImageId pullImage.getImage()
+
+                hostConfig.capAdd = ['NET_ADMIN']
+                hostConfig.capDrop = ['CHOWN']
+            }
+        """
+
+        String containerInspect = '''
+            task inspectContainer(type: DockerInspectContainer) {
+                dependsOn startContainer
+                finalizedBy removeContainer
+                targetContainerId startContainer.getContainerId()
+
+                onNext { c ->
+                    if(c.hostConfig.capAdd.size() != 1) {
```

Comment:
Let's use the `empty` method here.

Reply:
this is an array, not a list, so there is no empty() method on it, unless I'm thinking of the wrong method

Reply:
Got it. Forgot that we are checking a Docker Java type here.